src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SELECTED_ALFABET = 'Sort alphabetically';
+const SELECTED_LENGTH = 'Sort by length';
```

Comment:
Consider using consistent naming conventions for constants. The constant `SELECTED_ALFABET` seems to have a typo and should be `SELECTED_ALPHABET` for clarity and consistency.